app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt

```diff
@@ -49,6 +51,11 @@ class FavoriteFragment : Fragment() {
             thisActivity.supportActionBar?.setDisplayHomeAsUpEnabled(false)
         }
 
+        rootView.findSomethingToDo.setOnClickListener {
+            val bundle = Bundle()
+            Navigation.findNavController(rootView).navigate(R.id.searchFragment, bundle, Utils.getAnimFade())
```

Comment:
Why are you passing empty bundle?

Reply:
No need to use 'Navigation.'. Directly use `findNavController` as it is already imported.